lib/common/providers/charge_provider.dart

```diff
@@ -264,15 +331,45 @@ class CEVChargeProvider with ChangeNotifier {
       return;
     }
 
-    setStatus(LoadingStatus.loading);
+    var otherSignatories = [_station.address!];
```

Comment:
Do we deal with the below scenario in this modification? Or in our requirement, we don't need to worry about this.

1. UserA connects to stationA, then the extrinsic of sending/refunding is pending.
2. UserA connects to stationB, and shows the pending extrinsic for the stationA?

Reply:
There's not separation of StationA or B. We are just saving the transaction received from ServiceDelivered event. 
1. We save the transaction to local db.
2. We fetch the transaction from db on next app opening if the app was existed without approving transaction.
3. We delete transaction from local db only after successful approval.